client/src/components/Forums/ThreadsDisplay/ThreadsDisplay.tsx

```diff
@@ -77,6 +83,8 @@ const ThreadsDisplay = ({ forumId, onThreadSelect }: ThreadsDisplayProps) => {
             <small>
               Started by {thread.user.firstName} on{' '}
               {new Date(thread.createdAt).toLocaleDateString()}
+              <span className="mx-2 text-gray-400">|</span>
+              <span className="italic">{formatReplies(thread.postCount)}</span>
```

Comment:
⛏️ feel free to ignore this but I like declaring as few helper functions as possible to save from cluttering up the component. Personal preference tho